RecoHGCal/TICL/plugins/TrackstersMergeProducer.cc

```diff
@@ -528,9 +539,34 @@ void TrackstersMergeProducer::produce(edm::Event &evt, const edm::EventSetup &es
     }
   }
 
+  // Compute timing
+  computeTime(*resultCandidates);
+
   evt.put(std::move(resultCandidates));
 }
 
+void TrackstersMergeProducer::computeTime(std::vector<TICLCandidate> &resultCandidates) {
+  for (auto &cand : resultCandidates) {
+    if (cand.tracksters().size() > 1) {  // For single-trackster candidates the timing is already set
+      auto time = 0.;
```

Comment:
`auto` here means very little: it will be a `double` since you are assigning "0." to it.
```suggestion
      float time = 0.f;
      float timeErr = 0.f;
```

Reply:
Done:
https://github.com/cms-sw/cmssw/pull/32046/commits/0309768d2851d4cbfb13931fe980e57f2b33fe1a